docs/docs/reference/other-new-features/export.md

```diff
@@ -69,6 +69,7 @@ A member is _eligible_ if all of the following holds:
    a base class of the class containing the export clause.
  - it is accessible at the export clause,
  - it is not a constructor, nor the (synthetic) class part of an object,
+ - it is a constructor proxy
```

Comment:
That implies that *only* constructor proxies can be exported.

Reply:
Indeed it should have been appended to previous item as a note